apps/server/src/modules/webhook/webhook.controller.ts

```diff
@@ -17,7 +26,12 @@ export class WebhookController {
     @Param('sourceName') sourceName: string,
     @Body() eventBody: EventBody,
     @Res() response: Response,
+    @Query() { action }: { action?: string },
   ) {
+    if (eventBody.action === undefined) {
```

Comment:
@mrchocha is this generic for all webhooks or specific to sentry?

Reply:
currently its specific for sentry only.